build.sh

```diff
@@ -230,10 +230,10 @@ fi
 
 if [ "$DO_UPDATE" == true ] || [ ! -f $HTML_CACHE/w3cbugs.csv ]; then
   rm -f $HTML_CACHE/w3cbugs.csv
-  $QUIET || echo "Downloading list of W3C bugzilla bugs (can be a wee bit slow)..."
+  $QUIET || echo "Downloading list of W3C bugzilla bugs..."
   curl $($VERBOSE || echo "-s") \
     -o $HTML_CACHE/w3cbugs.csv \
-    'https://www.w3.org/Bugs/Public/buglist.cgi?columnlist=bug_file_loc,short_desc&query_format=advanced&resolution=---&ctype=csv&status_whiteboard=whatwg-resolved&status_whiteboard_type=notregexp'
+    'https://www.w3.org/Bugs/Public/buglist.cgi?columnlist=bug_file_loc,short_desc&query_format=advanced&resolution=---&ctype=csv&status_whiteboard=whatwg-resolved&status_whiteboard_type=notregexp&bug_file_loc=http&bug_file_loc_type=substring&product=WHATWG&product=HTML%20WG&product=CSS&product=WebAppsWG'
```

Comment:
That's the "bugs that have a URL" bit from the description. The URL field is `bug_file_loc` and I wanted it to be non-empty since that's what wattsi uses to match bugs to sections. Finding no direct was of checking for non-emptyness, I just checked if it contained "http" instead, which yields the intended results.